lib/__tests__/standalone-fix.test.mjs

```diff
@@ -205,7 +205,7 @@ describe('fix callback', () => {
 					},
 				}),
 			).rejects.toThrow(
-				'The fix callback for rule "plugin/selector-no-qux" requires index/endIndex or start/end or word to be passed to the `report()` function.',
+				'The fix callback for rule "plugin/selector-no-qux" requires a node to be passed to the `report()` function.',
 			);
 		});
```

Comment:
@romainmenke this change will allow the refactor you wanted between fix and report to happen.
Tell me, if you want to work on it after this PR.
Ill eventually do a small scale version of that refactor if you don't.

Reply:
#7769